File: prak_lat1 python/Processor.py
class Processor:

	# Constructor
	def __init__(self, name, price):
		self.name = name
		self.price = price

	# Set & Get methods
	def set_name(self, name):
		self.name = name


	def set_price(self, price):
		self.price = price


	def get_name(self):
		return self.name


	def get_price(self):
		return self.price

File: prak_lat1 php/Processor.php
<?php

class Processor{

	// Attributes
	private $nama;
	private $price;

	// Constructor
	public function __construct($nama, $price){
		$this->nama = $nama;
		$this->price = $price;
	}

	// Set & Get Methods
	public function setNama($nama){
		$this->nama = $nama;
	}

	public function setPrice($price){
		$this->price = $price;
	}

	public function getNama(){
		return $this->nama;
	}

	public function getPrice(){
		return $this->price;
	}
}
?>